src/StatsFuns.jl

```diff
@@ -233,10 +208,37 @@ export
     logmvbeta,          # logarithm of multivariate beta function
     lstirling_asym
 
+# reexports
+using LogExpFunctions
+using LogExpFunctions: loghalf, logtwo, logπ, log2π, log4π, softplus, invsoftplus
```

Comment:
@tpapp I guess it would be reasonable to export them in LogExpFunctions as well?